Mapping immudb data types to PostgreSQL.md

```diff
@@ -0,0 +1,64 @@
+# Mapping immudb data types to PostgreSQL  
+
+In this section, we take a look at the following:  
+
+-  For each immudb data type, which PostgreSQL is compatible?
+-  Is it possible to map more than one PostgreSQL data type to types in immudb?
+-  Size limits of PostgreSQL data types.
+
+The following are PostgreSQL data types compatible with immudb data types:
+
+## PostgreSQL numeric types
+
+immudb supports numeric types such as INTEGER. However, in PostgreSQL, there are many numeric types. [PostgreSQL 13.4](https://www.postgresql.org/docs/13/datatype-numeric.html) numeric types include such as  `SMALLINT`, `BIGINT`, `INTEGER`, `DECIMAL`, `SERIAL`, `SMALLSERIAL`, `BIGSERIAL`, `REAL`, `DOUBLE PRECISION`, and `NUMERIC`.
+
+It is possible for us to map the data type `INTEGER` in immudb to PostgreSQL `INTEGER` type.
+
+immudb `INTEGER` type is compatible with PostgreSQL `INTEGER` type. It is also possible to make use of the `SMALLINT` type.   
+
+Like other data types, the `INTEGER` type could also be used to define primary keys in the PostgreSQL table.
+
+Check this [page](https://www.postgresql.org/docs/13/datatype-numeric.html) for detailed information on PostgreSQL numeric types.
+
+## PostgreSQL conditional types
+
+**BOOLEAN**: PostgreSQL `BOOLEAN` has three states: `TRUE`, `FALSE`, and `UNKNOWN`. `UNKNOWN` in this context refers to SQL null value. The storage size for the `BOOLEAN` type is 1 byte.
+
+Both immudb and PostgreSQL support `TRUE` or `FALSE` as `BOOLEAN` values. However, immudb `BOOLEAN` type has a  third value known as `NULL` value. This `NULL` value refers to “missing” values just like PostgreSQL's `UNKNOWN` value.
+
+You can use the `BOOLEAN` type to define an entity state such as whether a user has subscribed to a premium product or not.
+
+Check this [page](https://www.postgresql.org/docs/current/datatype-boolean.html) for detailed information on PostgreSQL `BOOLEAN` type.
+
+## PostgreSQL binary  types    

```

Comment:
Good section, but can you add a summary table? Like, in the first column show immudb types, and in the second column the corresponding PostgreSQL types you mentioned in this page.